docs/stencil-docs/deploying-a-theme/theme-best-practices.md

```diff
@@ -0,0 +1,174 @@
+* [Largest Contentful Paint (LCP)](#largest-contentful-paint)
+* [First Input Delay (FID)](#first-input-delay)
+* [Cumulative Layout Shift (CLS)](#cumulative-layout-shift)
+
+### Largest Contentful Paint
+LCP is the metric that reports the time it takes to display the largest image on the screen. This metric is important because page speed ensures visitors receive the best performance and presentation of the site.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+**Good score:**
+The optimal LCP score is 2.5 seconds or faster. 
+
+</div>
+</div>
+</div>
+
+To improve LCP:
+* Make images smaller or remove anything preventing a quick download.
+* Avoid JavaScript or external scripts to load images.
+* Use an image CDN
+* Optimize your server
+
+### First Input Delay
+FID is the metric that measures the delay users experience when interacting with a page. It is the time it takes for the site to respond when a user clicks a link, selects a button, taps on the screen, etc.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+**Good score:**
+The optimal FID score is 100 milliseconds or less. 
+
+</div>
+</div>
+</div>
+
+To improve FID:
+* Reduce the effect of third-party code
+* Reduce JavaScript execution time
+* Reduce duration of your longest task
+
+Lighthouse does not display FID since it can only be measured in the field. Real user interaction is needed in order to measure the response delay. Instead, Lighthouse displays Total Blocking Time (TBT) which also captures a user's interactivity. Improvements to TBT will also improve your FID score.
+
+A good TBT score is 300 milliseconds or less.
+
+### Cumulative Layout Shift (CLS)
+
+CLS is the metric that measures visual stability. This metric measures how often users experience unexpected layout shifts. A layout shift is when a visible element changes its position from one rendered frame to the next. For example, a layout shift can occur when a user is viewing a page and a video moves on top of the home carousel. These unexpected movements can result in an unpleasant experience for the user.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+**Good score:**
+The optimal CLS score is 0.1 seconds or less. 
+
+</div>
+</div>
+</div>
+
+To improve CLS:
+* Add size attributes to elements
+* Reserve space for injected content
+* Avoid inserting ads or banners dynamically
+* Ensure to load critical CSS before content
+
+### Optimization example
+
+In this example, the performance score is currently 47. The LCP score is 6.2s, the TBT score is 410 ms, and the CLS score is 1.137 which are all less than optional scores.
+
+![Performance Metrics](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/performnce-example-1.png "Performance Metrics")
```

Comment:
What are these images ? They don't seem to load for me.

Reply:
Also there is a typo in the name of the file. Performance is missing an `a` in this one as well as the one below.